### ISSUE TITLE
[Docs page] Project Setup

### ISSUE THREAD

**Issue by schersh** [hashicorp/terraform-cdk#975]
**Description**

:tickets: [Asana ticket](https://app.asana.com/0/1200893566367079/1200893566367094)

:mag: File path: /website/docs/cdktf/create-and-deploy/project-setup-index.html.md

Project Templates: Built-in and [Remote](https://github.com/hashicorp/terraform-cdk/blob/main/docs/working-with-cdk-for-terraform/remote-templates.md
) 

Connect to Terraform Cloud ([page *needs more info](https://github.com/hashicorp/terraform-cdk/blob/main/docs/working-with-cdk-for-terraform/terraform-cloud.md))

- If you have TFC configured (have run Terraform Login in the past) we will use TFC when you create a new project (unless you opt out), if not we default to local
- If you have no TFC credentials we will also ask you whether you want to login

Project Structure 
-How CDKTF projects are structured - What files there are in a project with links to places to find out more information

Talk about convert option (cli command)

**Comment by laurapacilio** [comment]
@skorfmann put up an initial PR with edits and suggestions (thank you!). I reviewed and actually opened another PR to address those suggestions and copyedit the content that Sebastian already wrote here: https://github.com/hashicorp/terraform-cdk/pull/1058

his approach let me make edits and render the markdown on my local machine (so much easier) also let me do a few big tasks (like move the stacks stuff to a new page). My goal was to save Sebastian time and reduce the amount of back and forth we have to do through the github UI.

**Comment by schersh** [comment]
@laurapacilio is wondering if this content from the README belongs somewhere on this page? https://github.com/hashicorp/terraform-cdk/blob/main/packages/@cdktf/hcl2cdk/README.md#known-limitations

**Comment by laurapacilio** [comment]
yes! and I'm also wondering how this relates to the convert command. :-)

**Comment by skorfmann** [comment]
> @laurapacilio is wondering if this content from the README belongs somewhere on this page? https://github.com/hashicorp/terraform-cdk/blob/main/packages/@cdktf/hcl2cdk/README.md#known-limitations

This is the lib which powers the `cdktf convert` command. Don't think it belongs into the docs. However, a link might be worthwhile
